contracts/ToucanRegenBridge.sol

```diff
@@ -62,6 +78,8 @@ contract ToucanRegenBridge is Ownable, Pausable {
     constructor(address tokenIssuer_, INCTPool nctPool_) Ownable() {
         tokenIssuer = tokenIssuer_;
         nctPool = nctPool_;
+        _grantRole(PAUSER_ROLE, tokenIssuer);
+        _grantRole(PAUSER_ROLE, msg.sender);
```

Comment:
should only `msg.sender` be granted this role? or should it be just tokenIssuer? or both fine?

Reply:
I think both is fine.

Reply:
Btw since `tokenIssuer` may be empty, can you move that assignment inside the if clause below? Also use `tokenIssuer_` instead of `tokenIssuer` to avoid the storage read.

Reply:
Updated in https://github.com/regen-network/toucan-bridge/pull/47/commits/fd1a22da7a37be2ddaf07c0924e206db52579829